readthedocs/builds/forms.py

```diff
@@ -88,6 +89,8 @@ def save(self, commit=True):
         obj = super().save(commit=commit)
         if obj.active and not obj.built and not obj.uploaded:
             trigger_build(project=obj.project, version=obj)
+        if self.has_changed():
+            version_changed.send(sender=self.__class__, version=obj)
```

Comment:
Do we not want this on the model instead of the form? Seems like having it in the model `save` method is better, unless there's a reason we can't do it there.

Reply:
Models don't keep track of changed data. We could do this by overriding the init method of the models, but isn't guaranteed that the form will use the model in that way.